### PR TITLE
tkts parameters are true by default

### PR DESCRIPTION
**Pull request by brngylni**
This relates to issue #79. Parameters have been made True by default. Parameters themselves and also their descriptions are updated accordingly.

### CODE REVIEW

File: pagure_exporter/main.py [hunk 1]
```diff
@@ -118,43 +118,43 @@ def main(srce, dest, pkey, gkey, fusr, tusr):
...
 )
 @click.option(
-    "-c",
-    "--comments",
+    "-nc",
+    "--no-comments",
     "comments",
-    help="Transfer all the associated comments",
-    default=False,
+    help="Don't transfer the associated comments",
```
Comment (gridhead) [inline pagure_exporter/main.py:124]: Please revert this specific line to how it was before.

File: pagure_exporter/main.py [hunk 2]
```diff
@@ -118,43 +118,43 @@ def main(srce, dest, pkey, gkey, fusr, tusr):
...
 )
 @click.option(
-    "-a",
-    "--commit",
+    "-na",
+    "--no-commit",
     "commit",
-    help="Assert issue ticket states as they were",
-    default=False,
+    help="Issue ticket states don't have to be in the state they were in",
```
Comment (gridhead) [inline pagure_exporter/main.py:140]: Please revert this specific line to how it was before.

File: pagure_exporter/main.py [hunk 3]
```diff
@@ -118,43 +118,43 @@ def main(srce, dest, pkey, gkey, fusr, tusr):
...
 )
 @click.option(
-    "-t",
-    "--secret",
+    "-nt",
+    "--no-secret",
     "secret",
-    help="Confirm issue ticket privacy as they were",
-    default=False,
+    help="Don't account issue ticket privacy",
```
Comment (gridhead) [inline pagure_exporter/main.py:148]: Please revert this specific line to how it was before.

File: pagure_exporter/main.py [hunk 4]
```diff
@@ -118,43 +118,43 @@ def main(srce, dest, pkey, gkey, fusr, tusr):
...
 )
 @click.option(
-    "-o",
-    "--series",
+    "-mx",
+    "--mixed",
     "series",
-    help="Ensure issue ticket sequence as they were",
-    default=False,
+    help="Don't account the issue ticket sequence",
```
Comment (gridhead) [inline pagure_exporter/main.py:156]: Please revert this specific line to how it was before.

**Review by gridhead — CHANGES_REQUESTED** [review]
There is apparently some confusion at your end on what the options are supposed to achieve. Please make the requested corrections.

File: pagure_exporter/main.py [hunk 5]
```diff
@@ -99,7 +99,7 @@ def main(srce, dest, pkey, gkey, fusr, tusr):
     "status",
     type=click.Choice(["OPEN", "SHUT", "FULL"], case_sensitive=False),
     help="Extract issue tickets of the mentioned status",
-    default="OPEN",
+    default="FULL",
```
Comment (gridhead) [inline pagure_exporter/main.py:102]: The option which previously defaulted to transferring only OPEN tickets will now default to transferring tickets regardless of their states.

File: pagure_exporter/main.py [hunk 6]
```diff
@@ -118,43 +118,43 @@ def main(srce, dest, pkey, gkey, fusr, tusr):
...
 @click.option(
-    "-c",
-    "--comments",
+    "-nc",
+    "--no-comments",
     "comments",
-    help="Transfer all the associated comments",
-    default=False,
+    help="Don't transfer the associated comments",
+    default=True,
```
Comment (gridhead) [inline pagure_exporter/main.py:125]: The option which previously defaulted to not transferring comments alongside the issue tickets will now default to transferring comments too.

File: pagure_exporter/main.py [hunk 7]
```diff
@@ -118,43 +118,43 @@ def main(srce, dest, pkey, gkey, fusr, tusr):
...
 @click.option(
-    "-l",
-    "--labels",
+    "-nl",
+    "--no-labels",
     "labels",
-    help="Migrate all the associated labels",
-    default=False,
+    help="Don't migrate the associated labels",
+    default=True,
```
Comment (gridhead) [inline pagure_exporter/main.py:133]: The option which previously defaulted to not transferring labels alongside the issue tickets will now default to transferring labels too.

File: pagure_exporter/main.py [hunk 8]
```diff
@@ -118,43 +118,43 @@ def main(srce, dest, pkey, gkey, fusr, tusr):
...
 @click.option(
-    "-a",
-    "--commit",
+    "-na",
+    "--no-commit",
     "commit",
-    help="Assert issue ticket states as they were",
-    default=False,
+    help="Issue ticket states don't have to be in the state they were in",
+    default=True,
```
Comment (gridhead) [inline pagure_exporter/main.py:141]: The option which previously defaulted to not transferring issue states alongside the issue tickets will now default to transferring issue states too.

File: pagure_exporter/main.py [hunk 9]
```diff
@@ -118,43 +118,43 @@ def main(srce, dest, pkey, gkey, fusr, tusr):
...
 @click.option(
-    "-t",
-    "--secret",
+    "-nt",
+    "--no-secret",
     "secret",
-    help="Confirm issue ticket privacy as they were",
-    default=False,
+    help="Don't account issue ticket privacy",
+    default=True,
```
Comment (gridhead) [inline pagure_exporter/main.py:149]: The option which previously defaulted to not transferring issue privacy alongside the issue tickets will now default to transferring issue privacy too.

File: pagure_exporter/main.py [hunk 10]
```diff
@@ -118,43 +118,43 @@ def main(srce, dest, pkey, gkey, fusr, tusr):
...
 @click.option(
-    "-o",
-    "--series",
+    "-mx",
+    "--mixed",
     "series",
-    help="Ensure issue ticket sequence as they were",
-    default=False,
+    help="Don't account the issue ticket sequence",
+    default=True,
```
Comment (gridhead) [inline pagure_exporter/main.py:157]: The option which previously defaulted to not transferring the sequence of the issue tickets will now default to transferring the sequence too.

**Review by gridhead — COMMENTED** [review]
@brngylni here is a quick explanation of what was suggested to be done and how it affects the codebase.

**Review by gridhead — CHANGES_REQUESTED** [review]
Please fix the tests while you are at it.

File: pagure_exporter/main.py (hunk 1 above)
Comment (brngylni) [reply pagure_exporter/main.py:124]: Should it be False by default again or should I change just the description?

**Pull request closed without merging**